homeassistant/components/lektrico/translations/en.json

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
Please remove the translations folder. All translation strings go into `strings.json`.